components/application-authenticators/org.wso2.carbon.identity.application.authenticator.requestpath.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/requestpath/basicauth/BasicAuthRequestPathAuthenticator.java

```diff
@@ -121,6 +127,7 @@ protected void processAuthenticationResponse(HttpServletRequest request,
             if(log.isDebugEnabled()){
                 log.debug("BasicAuthentication failed while trying to get the tenant ID of the user " + username, e);
             }
+            audit.info(String.format(AUDIT_MESSAGE, username, "FAILURE"));
```

Comment:
Let's remove the audit logs from BasicAuthenticator because log in the framework, so it will get logged twice. But then in framework we don't log for error cases. We have to think about it further. Logging in each authenticator may not be scalable. So for now let's remove it.